### PR TITLE
[O2B-1134] Implement Runs per Simulation Pass view

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- Added Runs per Simulation Pass page

Notable changes for developers:
- Added runs filtering by simulationPassIds

Changes made to the database:
- NA

### CODE REVIEW

**Comment by xsalonx** [comment]
To be merged after https://github.com/AliceO2Group/Bookkeeping/pull/1373

**Comment by xsalonx** [comment]
To be merged after https://github.com/AliceO2Group/Bookkeeping/pull/1373

File: lib/public/views/Runs/RunsModel.js [hunk 1]
```diff
@@ -114,4 +117,23 @@ export class RunsModel extends Observable {
     get perDataPassOverviewModel() {
         return this._perDataPassOverviewModel;
     }
+
+    /**
+     * Load runs overview per simulation pass data
+     * @return {void}
+     */
+    loadPerSimulationPassOverview({ simulationPassId }) {
+        if (!this._perSimulationPassOverviewModel.pagination.isInfiniteScrollEnabled) {
```
Comment (martinboulais) [inline lib/public/views/Runs/RunsModel.js:126]: Why is this not in the load function?

File: lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewModel.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+export class RunsPerSimulationPassOverviewModel extends RunsOverviewModel {
+    /**
+     * Constructor
+     * @param {Model} model global model
+     */
+    constructor(model) {
+        super(model);
+        this._detectors$ = detectorsProvider.physical$;
+        this._detectors$.bubbleTo(this);
+        this._simulationPass = new ObservableData(RemoteData.notAsked());
```
Comment (martinboulais) [inline lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewModel.js:32]: Please suffix with $ the observable data

File: lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewPage.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        }), { profiles: ['runsPerSimulationPass'] }),
+    };
+
+    const commonTitle = h('h2', 'Runs per MC');
+    return h('', [
+        h('.flex-row.justify-between.items-center', [
+            h(
+                '.flex-row.g1.items-center',
+                simulationPass.match({
+                    Success: (payload) => breadcrumbs([commonTitle, h('h2', payload.name)]),
```
Comment (martinboulais) [inline lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewPage.js:58]: please put a proper name on payload, like `simulationPass` eventually renaming the current simualtionPass to `remoteSimulationPass`

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewModel.js (hunk 2 above)
Comment (xsalonx) [reply lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewModel.js]: changed

File: lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewPage.js (hunk 3 above)
Comment (xsalonx) [reply lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewPage.js]: applied

**Review by martinboulais — APPROVED** [review]

**Pull request merged**